src/App.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <header class="header">VOICEVOX Preview Pages</header>
+  <main class="main">
+    <p>プレビューするBranchまたはPull Requestを選択してください。</p>
+    <section class="downloads">
+      <template v-if="downloads">
+        <ElCard
+          v-for="download in downloads"
+          :key="download.dirname"
+          class="download-card"
+        >
+          <template #header>
+            <template v-if="download.source.type === 'branch'">
+              <ElTag effect="dark" type="primary" disableTransitions
+                >Branch</ElTag
+              >
+              <a
+                class="download-source"
+                :href="`https://github.com/VOICEVOX/voicevox/tree/${download.source.branch.name}`"
+                @click.stop
+              >
+                {{ download.source.branch.name }}
+              </a>
+              ：
+              <a
+                :href="`https://github.com/VOICEVOX/voicevox/commit/${download.source.branch.commit.sha}`"
+                @click.stop
+              >
+                {{ download.source.branch.commit.sha.slice(0, 7) }}
+              </a>
+            </template>
+            <template v-if="download.source.type === 'pullRequest'">
+              <ElTag effect="dark" type="success" disableTransitions>PR</ElTag>
```

Comment:
（めちゃめちゃ細かいですが）

広告に見えるかもしれない！（？）
弊害で言うと、広告ブロッカーが破壊してくるかもしれない（？）

ちょっと長いけどこっちでもいいかも（フォーマッターを動かさないといけないかも）
```suggestion
              <ElTag effect="dark" type="success" disableTransitions>Pull Request</ElTag>
```

Reply:
とは言っても自分の広告ブロッカーだと消えない + 最悪オフにすればいい、かつ長いと他の部分が読みづらくなるのでこのままにしようと思います。

Reply:
すみません、やっぱ略語は避けた方が良い気がしたので`Pull Request`にさせていただこうと思います 🙇 
どっちかというと避けたい、くらいの淡い気持ちですが、まあ。。